### PR TITLE
Albert's editing pass

### PR DESCRIPTION
**Pull request by ayshih**
Nothing too controversial, I believe.  I didn't include my edits on the introduction since that's still in flux (see #148).  I'll need someone else to remake Figures 1 and 4 for me because I can't actually build the figures on my machine.

### CODE REVIEW

File: sections/11-infrastructure.tex [hunk 1]
```diff
@@ -2,43 +2,44 @@ \subsection{Release Cycle, Versioning, and Long-Term Support}
...
+These services provide the first review of any contribution by running the test suite on each operating system (Windows, Mac, Linux), testing the documentation build, making comparison plots, and providing code-coverage metrics.
+Additionally, Travis CI\footnote{\url{https://travis-ci.org}} is used to run the entire test suite on a daily cadence to check for any changes in behavior due to changes in packages that \sunpypkg depends on.
 
 \subsection{Documentation and Gallery}
 \label{sec:docs}
 
-The \sunpyproj strives to provide up to date, approachable, and high quality documentation.
-All documentation for \sunpypkg, as well as all affiliated packages, is written using the commonly-used \code{Sphinx} documentation build system\footnote{\url{http://www.sphinx-doc.org/en/master/}}.
+The \sunpyproj strives to provide up-to-date, approachable, and high-quality documentation.
+All documentation for \sunpypkg, as well as all affiliated packages, is uses the commonly-used \code{Sphinx} documentation build system\footnote{\url{http://www.sphinx-doc.org/en/master/}}.
```
Comment (wtbarnes) [inline sections/11-infrastructure.tex]: "is uses the commonly-used" -> "uses the"

File: sections/3-support.tex [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-The \sunpyproj supports solutions like these for all relevant funding agencies \citep{nas_sunpy_white_paper} and furthermore has the ability to accept financial contributions from institutions or individuals through the NumFOCUS\footnote{\url{https://numfocus.org/}} organization. NumFOCUS is a 501(c)(3) public charity that collects and manages tax-deductible contributions for many open source scientific software packages such as \numpy and \astropy.
+The National Academies of Sciences, Engineering, and Medicine (NASEM) report on Open Source Software Policy Options for NASA Earth and Space Sciences \citep{NAP2018} outlines several solutions to alleviate these problems -- namely that the NASA Science Mission Directorate provide funding for new and existing open source software projects, promote scientists who spend time developing and improving open source software, and offer prizes for exemplary contributions to the open source software community.
```
Comment (wtbarnes) [inline sections/3-support.tex:8]: Add NASEM to the abbreviation table as well?

File: sections/4-development.tex [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
+	Major version releases are indicated along the top axis.
 	A striking reduction in the code base occurred after version 0.9.
 	This period saw a major code reorganization and deletion of obsolete features along with removing support for Python~2.
-	Middle panel: The cumulative number of committers (i.e. authors) to \sunpypkg as a function of time shows a steady increase in the number of people involved in the development team.
-	Right panel: A plot of the distribution of the number of commits per commiter.
-	This distribution indicates that the majority of commits are undertaken by a small group of committers. The average number of commits per committer is less than 10 commits.}
+	Middle panel: The cumulative number of contributors to \sunpypkg as a function of time shows a steady increase in the number of people involved in the development team.
+	Right panel: A plot of the distribution of the number of commits per contributor.
+	This distribution indicates that the majority of commits are undertaken by a small group of contributors. The average number of commits per contributors is less than 10 commits.}
```
Comment (wtbarnes) [inline sections/4-development.tex]: "contributors" -> "contributor" in the last sentence?

File: sections/9-affiliated.tex [hunk 4]
```diff
@@ -22,33 +22,34 @@ \subsection{drms}
 \label{sec:drms}
 
 The \package{drms} package provides functionality to access data hosted by the JSOC.
-Operated by Stanford, it is the primary data center for SDO HMI, SDO AIA, the Michelson Doppler Imager (MDI) aboard the \textit{Solar and Heliospheric Observatory} (SOHO) instrument, and the Interface Region Imaging Spectrograph (IRIS).
-This Data Record Management System (DRMS) is based on a PostgreSQL database that contains metadata, as well as pointers to image data, for every image contained in the archive.
-The \package{drms} package provides access to query the image metadata in the JSOC DRMS. It can also be used to submit tailored data export requests (e.g. movies and images in various formats) and download data files and is built on the HTTP/JSON interface provided by JSOC.
+Operated by Stanford, JSOC is the primary data center for SDO HMI, SDO AIA, the Michelson Doppler Imager (MDI) aboard the \textit{Solar and Heliospheric Observatory} (SOHO) instrument, and the \textit{Interface Region Imaging Spectrograph} (IRIS).
```
Comment (wtbarnes) [inline sections/9-affiliated.tex]: "Stanford" -> "Stanford University"

File: sections/9-affiliated.tex [hunk 5]
```diff
@@ -22,33 +22,34 @@ \subsection{drms}
...
-This package is planned to undergo major changes with new tools being developed in \code{astropy/specutils}\footnote{\url{https://github.com/astropy/specutils}}.
+This package is planned to undergo major changes to use new tools being developed in \code{astropy/specutils}\footnote{\url{https://github.com/astropy/specutils}}.
 
 \subsection{IRISPy}
 \label{sec:irispy}
 
-The \package{IRISPy} package provides tools to read, manipulate and visualize data from IRIS, \citep{DePontieu2014}.
-IRIS is a NASA Small Explorer mission comprised of two instruments: the Slit Jaw Imager (SJI) and a rastering slit spectrograph.
+The \package{IRISPy} package provides tools to read, manipulate and visualize data from IRIS \citep{DePontieu2014}.
+IRIS is a NASA Small Explorer mission comprises two instruments: the Slit Jaw Imager (SJI) and a rastering slit spectrograph.
```
Comment (wtbarnes) [inline sections/9-affiliated.tex]: Maybe "IRIS, a NASA Small Explorer mission, has two instruments: ..."

**Comment by ayshih** [comment]
@wtbarnes Thanks for the comments and catching the garbled edits!  I added NASEM to the abbreviation table, but we may need to re-balance what's actually included in that table because I think a bunch don't occur often enough to be considered "common", and there are others (e.g., RHESSI or STEREO) that aren't in the table.

**Comment by wtbarnes** [comment]
Agreed on the abbreviation table. When doing my abbreviation edit, my policy was to put it in the table if we used it more than once in the paper. If there are cases where that's not true, then it is a mistake on my part. There may also be abbreviations that are just not important enough to take up room there.

RHESSI and STEREO are only used in the Map/Timeseries table (I think?) so I did not include them in the abbreviations table.

**Comment by ayshih** [comment]
RHESSI and STEREO (as well as NoRH, LYRA, EVE, etc.) are mentioned in exactly one other place beyond that Map/TimeSeries table.  Even so, they may not be "common" enough to warrant being listed in the abbreviation table.

Conversely, HCC, SPoCA, and TAI are mentioned only once each in the text, and should probably be removed from the abbreviation table.

**Comment by wtbarnes** [comment]
Ah ok. HCC, SPoCA, and TAI were my mistake. I think it is sensible to remove the infrequently used instrument abbreviations you mentioned as well.

**Review by ehsteve — APPROVED** [review]
Thanks for the very thorough review @ayshih!

**Pull request merged**